scripts/sct_deepseg_lesion.py

```diff
@@ -51,14 +51,14 @@ def get_parser():
                       mandatory=True,
                       example=['t2', 't2_ax', 't2s'])
     parser.add_option(name="-centerline",
-                      type_value="multiple_choice",
```

Comment:
you want to keep "multiple_choice", otherwise the values "cnn" and "svm" will be considered as a nifti_input and the program will fail after parser checking.

Reply:
related to https://github.com/neuropoly/spinalcordtoolbox/pull/2020#discussion_r216363748

Reply:
My answer here: https://github.com/neuropoly/spinalcordtoolbox/pull/2020#discussion_r216497052